client/lib/analytics/ad-tracking/google-analytics-4.ts

```diff
@@ -1,41 +1,49 @@
 import isJetpackCloud from 'calypso/lib/jetpack/is-jetpack-cloud';
-import { GaPurchase } from '../utils/jetpack-cart-to-ga-purchase';
-import { GaItem } from '../utils/jetpack-product-to-ga-item';
+import { GaPurchase } from '../utils/cart-to-ga-purchase';
+import { GaItem } from '../utils/product-to-ga-item';
 import { TRACKING_IDS } from './constants';
 
 // Ensure setup has run.
 import './setup';
 
 export function setup( params: Gtag.ConfigParams ) {
-	// TODO: GA4 properties for WPCOM will be here
+	window.gtag( 'config', TRACKING_IDS.wpcomGoogleGA4Gtag, params );
 
 	if ( isJetpackCloud() ) {
 		window.gtag( 'config', TRACKING_IDS.jetpackGoogleGA4Gtag, params );
 	}
 }
 
-export function fireJetpackEcommercePurchase( purchase: GaPurchase ) {
+export function fireEcommercePurchase( purchase: GaPurchase, shouldSendToJetpack = false ) {
```

Comment:
Having `shouldSendToJetpack` as a boolean flag feels like a code smell and a limitation. Right now we have 2 entities to record GA for - Jetpack and WP.com, but what will happen if we have 3? With that in mind, isn't it better to have this argument accept a string (`wpcom` or `jetpack`)? That would allow adding more entities to track analytics in the future, if necessary.

Reply:
I second that; it sounds like a good idea. Especially since the code is written in Typescript and we can explicitly define enums.